ui/src/App.vue

```diff
@@ -0,0 +1,28 @@
+<template>
```

Comment:
Add a comment to clirfy that this is just a demo that I expect to be replaced b4 we use this.